packages/oui-inline-adder/README.md

```diff
@@ -0,0 +1,167 @@
+# oui-inline-adder
+
+<component-status cx-design="complete" ux="complete"></component-status>
+
+## Installation
+
+```less
+@import 'oui-inline-adder/inline-adder';
+```
+
+## Usage
+
+### Normal
+
+```html:preview
+<div class="oui-inline-adder">
+  <div class="oui-inline-adder__item">
+    <div class="oui-inline-adder__row">
+      <div class="oui-inline-adder__cell oui-inline-adder__cell-auto-grow">
```

Comment:
`auto-grow` is a modifier so it should be `_auto-grow`.

Reply:
This is done :)